pydicom/data/data_manager.py

```diff
@@ -284,7 +284,7 @@ def get_palette_files(pattern: str = "**/*") -> List[str]:
 
 
 def get_testdata_file(
-    name: str, read: bool = False
+    name: str, *, read: bool = False, download: bool = True,
```

Comment:
Making `read` a keyword only argument changes the existing behavior, so we at least have to document it in the [release notes](https://github.com/pydicom/pydicom/blob/master/doc/release_notes/v2.3.0.rst) under "Changes". Also, can you please add an entry for this feature under "Enhancements"?

Reply:
I think the existing behavior should not be changed.  If desired, keyword-only could apply to the new argument, but I don't think it is necessary to enforce that.

Reply:
I changed it because the changelog says it's keyword:

```
Added the `read` keyword parameter.
```